Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageBubbleView_Tests.swift

```diff
@@ -0,0 +1,297 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import XCTest
+
+final class ChatMessageBubbleView_Tests: XCTestCase {
+    // MARK: - Appearance
+
+    func test_appearance() {
+        // Create a bubble
+        let bubble = ChatMessageBubbleView().withFixedSize
+
+        // Assert the bubble is rendered correctly
+        AssertSnapshot(bubble, variants: .onlyUserInterfaceStyles)
+    }
+
+    func test_appearance_whenEphemeralMessageSentByCurrentUserLastInSequence() {
```

Comment:
The snapshots are not rendered correctly when `layer.maskedCorners` is set to round only some corners, not all of them 🙁 That's the key thing the message bubble does. Here is the issue: https://github.com/pointfreeco/swift-snapshot-testing/issues/358